app/(pages)/orders/order-cards/page.tsx

```diff
@@ -3,6 +3,60 @@ import styles from './page.module.scss';
 
 export default function ViewOrderCards() {
   const dateTime = new Date('2024-03-01T10:36:01.516Z');
+  const progressList: [string, Date, number, string, number][] = [
+    [
+      'Lightweight Water-Resistant Hooded Puffer Coat',
+      dateTime,
+      3,
+      '/sample-product/puffer.png',
+      0,
+    ],
+    [
+      'Lightweight Water-Resistant Hooded Puffer Coat',
+      dateTime,
+      1,
+      '/sample-product/watch.png',
+      2,
+    ],
+    [
+      'Lightweight Water-Resistant Hooded Puffer Coat',
+      dateTime,
+      1,
+      '/sample-product/sneaker.png',
+      1,
+    ],
+    [
+      'Lightweight Water-Resistant Hooded Puffer Coat',
+      dateTime,
+      1,
+      '/sample-product/plant.png',
+      0,
+    ],
+  ];
+  const deliveredList: [string, Date, number, string, number][] = [
```

Comment:
Probably have this data come in as an array of JSON objects rather than an array of arrays. This way you can query by field name rather than index